context.go

```diff
@@ -106,7 +107,9 @@ type Context struct {
 	globs    map[string]GlobPath
 	globLock sync.Mutex
 
-	fs pathtools.FileSystem
+	fs                 pathtools.FileSystem
+	sourceFinder       func(baseDir string) ([]string, error)
+	ExtraNinjaFileDeps []string
```

Comment:
Why not just returns a list of deps from SetModuleListFile like all of the other ctx functions?

Reply:
thanks